www/ui_components/add-server-view.html

```diff
@@ -93,6 +94,7 @@
       #server-config {
         padding: 24px;
       }
+      /* rtl:begin:ignore */
```

Comment:
Is it just the `direction` in line 113 you want to keep as `ltr`? Maybe you can reduce the size of this ignore block?https://www.npmjs.com/package/postcss-rtl#ignoring-specific-declarations